docs/tutorials/building-your-first-bot.mdx

```diff
@@ -0,0 +1,342 @@
+
+Copy and paste the following code into this file. This component allows us to drag and drop a prefab for our bots and
+a spawn point directly within our editor, and informs our SDK of these via those overridden functions.
+
+See the [full reference](/integrating-with-unity/seating-and-spawning-bots) for all the ways you can modify 
+the spawning of bots. Sometimes you may need to add this
+bot to a multiplayer network protocol, configure information like the character type, or define special start logic.
+This can all be done through the interface provided by `RGBotSpawnManager`.
+
+```cs
+using UnityEngine;
+
+using RegressionGames;
+using RegressionGames.Types;
+
+public class CharacterBotSpawnManager : RGBotSpawnManager
+{
+    
+    // Expose a field in the editor for the prefab to spawn as a bot
+    [SerializeField]
+    [Tooltip("The character to spawn")]
+    private GameObject rgBotPrefab;
+
+    // Expose a field in the editor to define where the bot should spawn
+    [SerializeField]
+    [Tooltip("Spawn point for RG Bots")]
+    private Transform botSpawnPoint;
+
+    public override GameObject SpawnBot(bool lateJoin, BotInformation botInformation)
+    {
+        // Loads the bot prefab and spawns it into the scene
+        var bot = Instantiate(rgBotPrefab, Vector3.zero, Quaternion.identity);
+        bot.transform.position = botSpawnPoint.position;
+        return bot;
+    }
+    
+    void Start()
+    {
+        // Prepares the RG SDK for spawning bots
+        RGBotServerListener.GetInstance()?.StartGame();
+        RGBotServerListener.GetInstance()?.SpawnBots();
+    }
+
+    private void OnDestroy()
+    {
+        // Stops all bots and cleans up the RG SDK
+        RGBotServerListener.GetInstance()?.StopGame();
+    }
+
+}
+```
+
+Once this component is implemented, create an empty object within the scene, rename it to `RGBotSpawner` (just so we can find it later),
+and attach this `CharacterBotSpawnManager` component to that object. Position it within the scene wherever you want to spawn your bot 
+(**make sure to set the Y coordinate to 0 so your bot does not float!**), and drag `RGBotSpawner`'s own Transform into the 
+**Bot Spawn Point** entry of the RGBotSpawner's `CharacterBotSpawnManager` component. Finally, find the 
+**Player** prefab provided by the sample project (located within `Assets/FirstBotDemo/Demo/Prefabs/Player.prefab`), and drag 
+this into the **Rg Bot Prefab** entry. Regression Games now knows how to load and unload bots from your scene! Make sure to save the 
+scene.
+
+![Screenshot of the RGOverlayCanvas.](./img/building-your-first-bot/spawner_6.png)
+
+## Add state information to relevant GameObjects
+
+We are onto the final parts of the integration! We need to indicate the following within our game:
+
+* What game state our bots can see
+* What actions our bots can perform
+
+First, let's add the state information. Regression Games marks states for bot consumption using the 
```

Comment:
```suggestion
First, let's define some state information. Regression Games marks states for bot consumption using the 
```

Reply:
And for following sentence, suggestion:

"Regression Games uses the `[RGState]` attribute to designate which state information bots can consume"